Kudu.Services/Functions/FunctionController.cs

```diff
@@ -179,6 +181,21 @@ public HttpResponseMessage Delete(string name)
             }
         }
 
+        [HttpGet]
+        public HttpResponseMessage DownloadFunctions(bool includeCsproj = true, bool includeAppSettings = false)
+        {
+            var tracer = _traceFactory.GetTracer();
+            using (tracer.Step($"{nameof(FunctionController)}.{nameof(DownloadFunctions)}({includeCsproj}, {includeAppSettings})"))
```

Comment:
right. The current proposal is on https://github.com/Azure/azure-functions-ux/issues/985. it looks like this
![image](https://cloud.githubusercontent.com/assets/4260261/24123781/203d2dec-0d7e-11e7-8e23-f71934d94b6c.png)